coral/package.json

```diff
@@ -18,6 +18,8 @@
     "reformat": "prettier --write src",
     "test": "jest",
     "test-dev": "jest --watch",
+    "typecheck": "tsc --noEmit --pretty",
```

Comment:
I think this command can be removed as `tsc` was introduced in 2fcb6286f18663adef8e858ddace38c1451af924